sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -176,6 +176,9 @@ object IntervalUtils {
   private val dayTimePatternLegacy =
     "^([+|-])?((\\d+) )?((\\d+):)?(\\d+):(\\d+)(\\.(\\d+))?$".r
 
+  private val fallbackNotice = ", set spark.sql.legacy.fromDayTimeString.enabled to true " +
```

Comment:
Please, use `${SQLConf.LEGACY_FROM_DAYTIME_STRING.key}`

Reply:
Thanks, done in 7140a71